community/views.py

```diff
@@ -1,17 +1,106 @@
 from django.shortcuts import render
-
 import json
 
 from django.http import JsonResponse
 from django.views import View
 
-from .models import PostImage, Post, Hashtag
+from .models import HouseSize, HouseStyle, HousingType, Space,  Post, PostBlock
+#from user.utils import login_decorator
+from product.models import *
+from user.models import *
+
+class Posting(View):
+
+    #@login_decorator
+    def post(self, request):
+        data = json.loads(request.body)
+
+        # 토큰 전까지 사용할  유저 아이디
+        user_id      = 2
+
+        house_size   = data.get('house_size')
+        house_style  = data.get('house_style')
+        housing_type = data.get('housing_type')
+        block        = data['block']
+
+        try:
+            Post.objects.create(
+                user_id=user_id,
+                house_size_id=house_size,
+                house_style_id=house_style,
+                housing_type_id=housing_type
+            )
+
+            post = Post.objects.all().last()
+
+            for i in block:
+                PostBlock.objects.create(
+                    image=i["image"],
+                    content=i["content"],
+                    space_id = i["space"],
+                    post_id=post.id,
+                )
```

Comment:
```suggestion
           post_block = [ 
                PostBlock(
                    image=item["image"],
                    content=item["content"],
                    space_id = item["space"],
                    post_id=post.id,
                ) for item in block
            ] 
            PostBlock.objects.bulk_create(post_block)
```
반복문 구현시 i 같은 변수 사용은 지양해 주세요!
블록 갯수만큼 생성하시는 거라면 bulk_create()를 사용해주세요. 
https://docs.djangoproject.com/en/3.1/ref/models/querysets/#bulk-create